snooty/parser.py

```diff
@@ -517,6 +518,26 @@ def handle_directive(
                 self.diagnostics.append(Diagnostic.error(msg, util.get_line(node)))
         elif name == "toctree":
             doc["entries"] = node["entries"]
+        elif domain == "devhub" and name == "author":
+            image_argument = options.get("image", None)
```

Comment:
The second argument of `dict.get()` actually defaults to None, so you don't need to include that